### ISSUE TITLE
Deprecated division

### ISSUE THREAD

**Issue by brendon** [justboil/bulma-checkbox#1]
Hi there, I'm getting this warning when compiling :)

```
DEPRECATION WARNING: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(str-length($svg), $slice) or calc(str-length($svg) / $slice)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
17 │     $loops: ceil(str-length($svg)/$slice);
   │                  ^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    node_modules/bulma-checkbox/scss/_functions.scss 17:18  svg-encode()
    node_modules/bulma-checkbox/scss/_functions.scss 40:13  checkmark()
    node_modules/bulma-checkbox/scss/_checkbox.scss 34:59   @import
    node_modules/bulma-checkbox/bulma-checkbox.scss 3:9     @import
    src/scss/global.scss 2:9                                root stylesheet
```

**Comment by vikdiesel** [comment]
Fixed in v1.2.0

**Comment by brendon** [comment]
Thanks @vikdiesel :)
